src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -141,7 +161,7 @@ export default class DefaultBrowserBehavior implements BrowserBehavior {
   }
 
   screenShareUnsupported(): boolean {
-    if (this.isSafari()) {
+    if (this.isSafari() || this.isIOSChrome() || this.isIOSFirefox()) {
```

Comment:
This is for the legacy screenshare so we should remove it instead.

Reply:
I just put a todo comment to remove it in the next major version update, in case anyone might be using it.